### ISSUE TITLE
Screenreader: Login screen username label doesn't exist

### ISSUE THREAD

**Issue by nicklee** [wagtail/wagtail#7432]
### Issue Summary

When trying to login to wagtail using JAWs the username field doesn't have a label, making it tricky to understand what's required

### Steps to Reproduce

1. Load a wagtail admin login screen in JAWS (/admin usually)
2. Navigate to the username field
3. Only "Edit" will be announced for the input field, 

This should be "Enter username"

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes)

Image showing what's displayed in JAWs:
![image (2)](https://user-images.githubusercontent.com/298766/129183911-5589c3f3-a8e6-4e1b-8f2a-2b8c51e1aebd.png)

Inspector in Chrome/Firefox reports that the labels don't meet WCAG requirements - they seem to be display: none at the moment:
![image (3)](https://user-images.githubusercontent.com/298766/129183909-faf032fa-9dfb-4e4a-a701-3c809e811d00.png)

**Comment by lb-** [comment]
Fixed via f1b3bbc52f2944dde0a37d1c33b47f44d03d4a76 & #7468
